package.json

```diff
@@ -71,7 +71,7 @@
     "lodash": "~4.17.5",
     "logops": "2.1.0",
     "mkdirp": "~0.5.1",
-    "mongodb": "~2.2.35",
+    "mongodb": "~3.6.12",
```

Comment:
CNR entry missing. Something like this:

```
- Upgrade: mongodb dependence from ~2.2.35 to ~3.6.12
```

Reply:
Done in https://github.com/telefonicaid/fiware-sth-comet/pull/568/commits/7f6f169c8aeca97cf4e51e07eb2d9e06eb4394f9